spec/support/oauth_helpers.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module OAuthHelpers
+  def token_request_params
+    @user ||= Fabricate(:user)
```

Comment:
@malparty I guess we can use fabricators instead of creating params for these. For fabricators, we can also create them without a class. Refer to this example in the1-api. 

https://github.com/nimblehq/the1-api/blob/development/spec/fabricators/v2/token.rb

Reply:
This one is not extremely important, but according to your preference :)

Reply:
When fabricating a token, can it be valid? 
From my tests, revoking a fabricated tokens result in an error as that stub is not valid. 

So I'm not sure where exactly we could use a stub:
- test: 'ask for a token' = we are actually testing to query a token
- test 'when a user refreshes an existing token on time' = we need a valid token
- test 'when a user revokes an existing token' = same

Or maybe I did not understand something ~

Reply:
@malparty In this my idea was this, you do not fabricate the token, but only the params. Then in test you can get params from this fabricator. The main advantage here is that we will not need to use ruby merge, and can use fabricator syntax to override any param. But as i said, its optional and does not provide a big benefit 👍 

```
    Fabricate(:token_params, from: 'OpenStruct') do
    end
```

```
    params = Fabricate(:token_params).to_h
```

Reply:
Oh, okay now I understand the idea!! 
I will try to apply it for the next features if it's ok for you!! :) 
--> I did not know we can override params with the Fabricate(..) method directly, thanks!!!